doc/source/language/syntax.rst

```diff
@@ -41,7 +66,28 @@ operator symbols:
 
 **Other Symbols**::
 
-    ( )  {  }  [  ]  ,  :  #  @  //
+    * ``//`` -> Comment indicator - starts a comment that runs until
+      the end of the line.
+    * ``( )`` -> Used either to group expressions to change the order of
+      operations (in the usual fashion), or to mark the parameters of
+      a function being called, like ``min(a, b)``.
+    * ``{  }`` -> Used to group a list of statements together into a
+      single block of statements.  Used to define the beginning and
+      ending of a function body, loop body, or conditional body.
+    * ``[  ]`` -> Denotes a list index or lexicon index.
+    * ``#`` -> An archaic way to index lists that is mostly supplanted by
+      the ``[ ]`` square bracket operator, but exists for backward
+      compatibility.
+    * ``,`` -> Separates arguments when calling a function, or parameters
+      when defiing a function.
```

Comment:
There's a typo here - 'defiing' missing an 'n'.

Reply:
Thanks.  I fixed it in a PR-less naked commit (not the best practice but I couldn't bring myself to make a new PR for just this.)